packages/frontend/src/pages/Home.jsx

```diff
@@ -56,22 +51,14 @@ export default observer(() => {
           <div className="info-grid">
             <InfoCard
               heading="Total Attesters/Apps"
-              value1={unirep.deploymentIds.length - 1}
-            />
-            <InfoCard
-              heading="Total Sign Ups"
-              value1={unirep.allSignUps.length}
+              value1={unirep.attesterCount}
```

Comment:
Yes, it's canonically correct and a valid attester. It's just impossible to sign up with it. Also the next epoch transition time is hard to display because it's 2^64